src/Compression/LZ4_decompress_faster.cpp

```diff
@@ -4,6 +4,7 @@
 #include <iostream>
 #include <Core/Defines.h>
 #include <Common/Stopwatch.h>
+#include <Common/CpuId.h>
```

Comment:
This header is unused. But should be used :)

Reply:
So keep it here is OK?

Reply:
This header removed after using `TargetSpecific`